src/ui/controls/shortcut/index.tsx

```diff
@@ -88,11 +89,16 @@ export default function ShortcutLink(props: ShortcutLinkProps) {
         });
     }
 
+    function onUpdate(props) {
+        props.textContent = shortcutMessage;
+    }
+
     return (
         <a
             class={cls}
             href="#"
             onclick={onClick}
+            updated={onUpdate}
         >{props.textTemplate(shortcut)}</a>
```

Comment:
The problem was in this line, it creates a string and a virtual text node is created for it. But since we use real DOM manipulations, let's make text empty. We can just leave it as `<a ... />`.